{{ cookiecutter.hosting_github_repo_name }}/SETUP.md

```diff
@@ -0,0 +1,122 @@
+
+9.  Create (or sign in to) a Pulumi account.
+    ([Pulumi Account Sign-Up](https://app.pulumi.com/signup))
+
+    Pulumi tracks and manages the state of your instance's infrastructure
+    (databases, file storage servers, credentials, etc.). More details can be found
+    [here](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment#pulumi).
+
+10. Create a [Pulumi Access Token](https://app.pulumi.com/account/tokens).
+
+    -   Click "Create token".
+    -   Set the description to: "{{ cookiecutter.municipality_slug }}-github-actions-runner".
+    -   Click "Create token".
+    -   Copy and save the token somewhere safe. We will use it later.
+        _Do not save this token as a file in this repository's directory._
+
+11. Create the GitHub repository for this deployment to live in.
+    [Create a new Repository](https://github.com/new)
+
+    -   Set the repo name to: **{{ cookiecutter.hosting_github_repo_name }}**
+    -   Set the repo owner to: **{{ cookiecutter.hosting_github_username_or_org }}**
+    -   Set the repo visibility to: "Public"
+    -   Do not initialize with any of the extra options
+    -   Click "Create repository".
+
+12. Initialize and push the local repository to GitHub.
+
+    -   In a terminal, while in this repository's directory, run:
+        ```bash
+        git init
+        git add -A
+        git commit -m "Initial commit"
+        git branch -M main
+        git remote add origin {{ cookiecutter.hosting_github_url }}.git
+        git push -u origin main
+        ```
+    -   Refresh your repository's dashboard to ensure that all files were pushed.
+
+13. Configure [repository settings]({{ cookiecutter.hosting_github_url }}/settings).
+
+    -   In the "Options" tab, configure "GitHub Pages"
+        -   Select "gh-pages" from the "Source" dropdown.
+        -   Click "Save".
+    -   In the "Secrets" tab, configure two "Actions secrets"
+        -   Click "New repository secret".
+        -   Set "Name" to: **GOOGLE_CREDENTIALS**
+        -   Open up the file downloaded from Google Cloud (the Service Account JSON),
+            copy all text in the file, paste the text into the "Value" field.
+        -   Click "Add secret".
+        -   Click "New repository secret".
+        -   Set "Name" to: **PULUMI_ACCESS_TOKEN**
+        -   Open up the file or browser tab where you saved your Pulumi token,
+            copy the token, past the token into the "Value" field.
+        -   Click "Add secret".
+
+14. Navigate to [GitHub Actions History]({{ cookiecutter.hosting_github_url }}/actions).
+
+    -   Click on the "workflow run" tagged with "Initial commit" and "Infrastructure".
+    -   Click the "Re-run jobs" dropdown.
+    -   Click "Re-run all jobs".
+
+**If all steps complete successful your web application will be viewable at: {{ cookiecutter.hosting_web_app_address }}**
+
+Once all of these steps are done, feel free to delete this section of the README.
```

Comment:
This is a separate file now. No need to delete!

Reply:
Lol oh whoops, thanks for the catch.